PROJECT_CHARTER.md

```diff
@@ -0,0 +1,72 @@
+# WebdriverIO Charter
+
+## Section 0: Guiding Principles
+
+The WebdriverIO project is part of the OpenJS Foundation which operates transparently, openly, collaboratively, and ethically. Project proposals, timelines, and status must not merely be open, but also easily visible to outsiders.
+
+## Section 1: Scope
+
+The project tries to provide users a variety of building blocks to create a highly customizable, maintainable and scaleable framework to build and run end-to-end (e2e) tests for web and mobile applications. While it is focused on running these tests in accordance with the [WebDriver](https://w3c.github.io/webdriver/) standard, it also leverages other interfaces to provide as much control of the automated user agent as possible.
```

Comment:
`scaleable -> scalable`

Reply:
maybe change `end-to-end (e2e) tests` to `tests` because people can also run UI/integration tests for example. it's debatable though :)

Reply:
Technically yes, but I think this section should describe how this project is intended to be used.